.github/workflows/buf-pull-request.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Check for breaking changes
         uses: bufbuild/buf-breaking-action@v1
         with:
-          input: 'crates/proto'
+          input: 'proto'
           # The 'main' branch of the GitHub repository that defines the module.
           against: 'https://github.com/${GITHUB_REPOSITORY}.git#branch=main,subdir=crates/proto'
```

Comment:
N.B. the `against` line should be updated post-merge, but is accurate as of right now, showing us potentially breaking changes in the CI output of this PR.